bgpd/rfapi/rfapi.c

```diff
@@ -3368,13 +3368,15 @@ DEFUN (debug_rfapi_query_vn_un_l2o,
 {
 	struct rfapi_ip_addr vn;
 	struct rfapi_ip_addr un;
+	int rc;
+#if 0 /* FIXME Broken feature */
```

Comment:
As with previous PR, I personally prefer removal of dead code.  @louberger opinions, since you care about this feature the most.

Reply:
@paulzlabn what do you think?